packages/pages/src/components/address/methods.test.ts

```diff
@@ -85,6 +98,12 @@ describe("getDirections()", () => {
     );
   });
 
+  it("returns URL to Google Maps GMB listing without being case sensitive", () => {
+    expect(getDirections(undefined, sampleListingsCaseInsensitive)).toEqual(
```

Comment:
I don't think this is exactly what we want. Our getDirections function could always be using the lowercase version, but since we pass in both we wouldn't know that based on this test case. 

I think we want an input where it's only lowercase, and one where it's only uppercase, and show that in both situations we get the listingUrl out.